globus_automate_client/cli/flows.py

```diff
@@ -898,6 +898,32 @@ def flow_action_status(
     ).run_and_render()
 
 
+@app.command("run-definition")
+def get_flow_definition_for_run(
+    run_id: str = typer.Argument(...),
+    flow_id: uuid.UUID = typer.Option(
+        ...,
+        help="The ID for the Flow which triggered the Action.",
+    ),
+    flow_scope: str = typer.Option(
+        None,
+        help="The scope this Flow uses to authenticate requests.",
+        callback=url_validator_callback,
+    ),
+    flows_endpoint: str = flows_env_var_option,
+    output_format: OutputFormat = output_format_option,
+    verbose: bool = verbosity_option,
+):
+    """
+    Get the flow definition and input schema used to start a run.
+    """
+    fc = create_flows_client(CLIENT_ID, flows_endpoint)
+    method = functools.partial(
```

Comment:
I wouldn't spend more than a handful of minutes on this if it is even possible, but given that the route doesn't require the flow ID nor permissions on the flow itself, do we need to solicit it in this command?

Reply:
No, we do not. This same requirement exists on all of the run-related routes, due to how the authorizers are set up (and I'm pretty sure I created a story about removing this requirement, but the priority continues to be to migrate to the Globus CLI).

I already didn't spend more than a handful of minutes on this, which is why this requirement still exists here. 😢😞